### PR TITLE
Only depends on the dependencies we need

### PR DESCRIPTION
**Pull request by rafaelfranca**
We don't need the whole framework as dependency so we don't need to put the rails meta.

I also added test for the gem to make sure it works with Rails 5.

Review @kmcphillips.

There is a test failing but it is failing also in master. I can fix it if you point the expected behavior.

### CODE REVIEW

**Comment by kmcphillips** [comment]
Let me look at the failing test first. I'll stay on top of this. Thanks @rafaelfranca!

File: lib/measured/rails/validations.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
+require 'active_model'
+require 'active_model/validations'
```
Comment (kmcphillips) [inline lib/measured/rails/validations.rb]: I'd prefer to see these in [base.rb](https://github.com/Shopify/measured-rails/blob/master/lib/measured/rails/base.rb), as base is always loaded and loads this file and, so it's not like these will be loaded lazily.

I prefer to be able to look in one place for the required dependencies, and not find them littered at the top of various class files.

**Comment by kmcphillips** [comment]
Test is fixed on master so you can rebase it into this branch. 

After that, one comment and the rest is :+1: :star:.

**Comment by rafaelfranca** [comment]
Done

**Comment by rafaelfranca** [comment]
Will you bump the version or should I use the git repository?

**Comment by kmcphillips** [comment]
I'll bump `measured` and `measured-rails` to `1.3.0` and bump the version in Shopify. Standby.

**Pull request merged**